media/src/objects/Function.svelte

```diff
@@ -308,7 +314,7 @@
 
     let surfaceMesh = new THREE.Object3D();
 
-    const updateSurface = function() {
+    const updateSurface = function () {
```

Comment:
We always define functions like this `function() {` rather than your change here. but, the styles aren't that strict in this project and I'll let this slide just to get your functionality changes in here.